### DISCUSSION TITLE
Autoscaler scaling in

### DISCUSSION THREAD

**Discussion by someshkoli** [kedacore/http-add-on#561] [General] [answered]
After how much time down this addon downscales the deployment ? is there any way to specify this in spec ? If not then would love to add this feature :thinking:

**Comment by someshkoli** [comment, marked as answer]
Can close this, found duplicates

**Reply by tomkerkhove** [reply]
For others, the duplicate was https://github.com/kedacore/http-add-on/discussions/550
